examples/src/main/java/org/bitcoinj/examples/ExamplePaymentChannelClient.java

```diff
@@ -79,7 +95,11 @@ public void run(final String host) throws Exception {
                 return ImmutableList.<WalletExtension>of(new StoredPaymentChannelClientStates(null));
             }
         };
-        appKit.connectToLocalHost();
+        // Broadcasting can take a bit of time so we up the timeout for "real" networks
+        final int timeoutSeconds = params == RegTestParams.get() ? 15 : 150;
```

Comment:
Unfortunately, RegTestParams.get() has side effects. If you want to test for specific params, can you use NetworkParams.getId()?